ledger-core/network/nds/msgdelivery/ctl_retry_state.go

```diff
@@ -54,6 +54,8 @@ func (p *stateSender) init(oobQueue, jobQueue int) {
 	p.oob = make(chan rqShipment, oobQueue)
 	p.states = make(chan stateJob, oobQueue)
 	p.jobs = make(chan retryJob, jobQueue)
+	p.requests = make(map[ShipmentID]rqShipment)
```

Comment:
Why do you do this?

Reply:
(*stateSender).put will not be working without map initialization.

Reply:
ok, my bad